tests/fixtures/basic-js/.sauce/config.yml

```
@@ -16,7 +16,7 @@ playwright:
 suites:
   - name: "saucy test"
     platformName: "Windows 10"
-    testMatch: '**/*.js'
+    testMatch: '.*.js'
```

In the real config, we can only support an array due to type safety.
```suggestion
    testMatch: ['.*.js']
```